dist/web3.js

```diff
@@ -1444,6 +1444,9 @@ web3.setProvider = function (provider) {
     this.currentProvider = provider;
     RequestManager.getInstance().setProvider(provider);
 };
+web3.isConnected = function(){
```

Comment:
The main and important point of this methods is that dapp developers have a reliable way to check if a node i connected, to e.g. show the user some popup exlpaining how to start a node. It has nothing to do with the provider per se, just the way to check if its running can be different per provider.